test/e2e/kibishii_tests.go

```diff
@@ -98,7 +106,7 @@ func GenerateData(ctx context.Context, namespace string, levels int, filesPerLev
 	return err
 }
 
-func VerifyData(ctx context.Context, namespace string, levels int, filesPerLevel int, dirsPerLevel int, fileSize int,
+func verifyData(ctx context.Context, namespace string, levels int, filesPerLevel int, dirsPerLevel int, fileSize int,
 	blockSize int, passNum int, expectedNodes int) error {
 	kibishiiVerifyCmd := exec.CommandContext(ctx, "kubectl", "exec", "-n", namespace, "jump-pad", "--",
```

Comment:
I can't comment on where the error is happening for me, but I am getting failures here because the first read from `stdoutReader` is `EOF`. I think this function should be changed to use `Run()` which runs the process and waits for it to finish, rather than calling `Start()` and reading line by line as the verify command runs. It doesn't seem like there's any specific reason to do it that way?

Reply:
I hadn't run into this error before and bc this code was just moved across files in this PR I had left it alone.
But yeah, we seem to only care if the command was successful or not and we also have a package at https://github.com/vmware-tanzu/velero/blob/main/pkg/util/exec/exec.go to do this. 
I've updated the code to use this package.

Reply:
Marking this comment as resolved.

Reply:
It was surprising that I hadn't run into this before.
But in this case, considering that we only want to know if or not the command is successful we can easily switch it out for the `.Run()`.
Also, we have a package that does this for us. https://github.com/vmware-tanzu/velero/blob/main/pkg/util/exec/exec.go Updated the code to switch that out.

Reply:
marking this as resolved.